phoenicis-scripts/src/main/java/org/phoenicis/scripts/nashorn/NashornScriptInterpreter.java

```diff
@@ -32,7 +33,30 @@ public NashornScriptInterpreter(NashornEngineFactory nashornEngineFactory) {
 
     @Override
     public void runScript(String scriptContent, Consumer<Exception> errorCallback) {
-        nashornEngineFactory.createEngine().eval(scriptContent, errorCallback);
+        final String language = Locale.getDefault().getLanguage();
+        final String i18nextInit = "i18next.init({                                                                     "
+                + String.format("  lng: '%s',                                                             ", language)
```

Comment:
I don't like to define code as a String.
This looks quite ugly and hard to maintain.
Is there another way to set this?